docs/AUTHENTICATION.md

```diff
@@ -0,0 +1,43 @@
+## Authentication
+Plio Analytics uses Auth0 authentication to manage access tokens and valid API requests from the client app, which in Plio's ecosystem is the frontend app.
```

Comment:
Are we using Secuity context?

Reply:
Right now we'll proceed without security context as we have machine-to-machine API calls between frontend and analytics.

The ACL checks will be handled by frontend/backend repos.